cdap-ui/app/features/adapters/controllers/list-ctrl.js

```diff
@@ -32,18 +37,24 @@ angular.module(PKG.name + '.feature.adapters')
       });
 
     function pollStatus(app) {
-      dataSrc.poll({
-        _cdapNsPath: '/adapters/' + app.name + '/status'
-      }, function(res) {
-        app.status = res.status;
-      });
+      var statusParams = angular.extend({
+        app: app.name
+      }, params);
+
+      myAdapterApi.pollStatus(statusParams)
```

Comment:
Could we check if the status is STOPPED and stop the poll? Could avoid polling during STOPPED duration (as the status is not going to change unless the user starts it manually)